bcgov/biohubbc
# SIMSBIOHUB-275: Remove Project Coordinator from SIMS

## Links to Jira Tickets

- https://apps.nrs.gov.bc.ca/int/jira/browse/SIMSBIOHUB-275

## Description of Changes
 - Removes any Project Coordinator UI for creating, editing or viewing projects
 - Removes any reference to Project Coordinator in the frontend and backend codebase
 - Adds a migration that makes project coordinator fields optional, retaining any existing project coordinator data.

## Testing Notes (for local environment)

1. On branch `dev`, create one or more projects.
2. Check out the `SIMSBIOHUB-275` branch and run `make web` to run the new migration.
3. [ ] Upon compiling the project, you should be able to:
 - View an existing project + participants
 - Edit an existing project
 - Create a new project
 - View a survey + participants
 - Add participants to a project
 - Add participants to a survey

4. [ ] For the projects created before running the migration, their participant data in the project table should still exist.
